.appveyor.yml

```diff
@@ -94,6 +94,7 @@ for:
       - rizin -v
       - copy C:\Python38-x64\python.exe C:\Python38-x64\python3.exe
       - python3 -m pip install "git+https://github.com/rizinorg/rz-pipe#egg=rzpipe&subdirectory=python"
+      - "%PYTHON%\\Scripts\\ninja -C build test"
```

Comment:
i may want to remove this for now and enable it in another PR. Unit tests on windows do not work well because of `test_rzpipe` that fails. The work here should help fixing rzpipe C, i hope.